cli/Cargo.toml

```diff
@@ -126,3 +126,22 @@ pkg-url = "{ repo }/releases/download/v{ version }/jj-v{ version }-{ target }.{
 
 [lints]
 workspace = true
+
+[package.metadata.deb]
+name = "jj"
+maintainer = "Your Name <your@email.com>"
```

Comment:
(Optional): perhaps change this to `No Dedicated Debian Maintainer <nobody@example.com>`?